test-adapter/src/main/scala/org/scalajs/sbttestadapter/TaskDefSerializers.scala

```diff
@@ -16,7 +16,7 @@ import org.scalajs.core.tools.json._
 import FingerprintSerializers._
 import SelectorSerializers._
 
-object TaskDefSerializers {
+private[scalajs] object TaskDefSerializers {
```

Comment:
Ah... I thought we haven't released this. But indeed, [the reverse change][1] is in v0.6.9... :-/
So I guess I shouldn't do this.

[1]: https://github.com/scala-js/scala-js/commit/1d775830780750217f99c14ea73340d4fa06b540#diff-36e6887d1a47947dbb3a572909fc3a6d is